torch/nn/utils/parametrize.py

```diff
@@ -697,6 +697,7 @@ def remove_parametrizations(
                 if type(original) is torch.Tensor:
                     _maybe_set(original, t)
                 else:
+                    assert isinstance(original, torch.Tensor)
```

Comment:
Can we more this after the getattr above and raise an error saying that the parametrization didn't return a Tensor as the error message?

Reply:
We already do a type is check?